scot/csp.py

```diff
@@ -59,7 +59,7 @@ def csp(x, cl, numcomp=None):
         sigma2 += np.cov(x2[t, :, :]) / x2.shape[0]
     sigma2 /= sigma2.trace()
 
-    e, w = eig(sigma1, sigma1 + sigma2, overwrite_a=True, overwrite_b=True, check_finite=False)
+    e, w = eigh(sigma1, sigma1 + sigma2, overwrite_a=True, overwrite_b=True, check_finite=False)
```

Comment:
Hehe, ok, but now we have to pollute this PR by discussing why this is necessary.

Why is it necessary? :)